### PR TITLE
Point users to CI build artifact

### PR DESCRIPTION
**Pull request by tiran**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,16 @@ Once you've built the Emscripten'd Python, you can rebuild it via
...
+
+* download the *wasm* build artifact bundle from the latest
+  [CI workflow](https://github.com/ethanhs/python-wasm/actions?query=branch%3Amain)
+  actions run.
+* unzip ``wasm.zip``
+* run a local webserver in the same directory as ``python.html``,
+   e.g. ``python3 -m http.server``
+* open http://localhost:8000/python.html
+* enter commands into the browser modal window and check the web developer
+  console (*F12*) for output
```
Comment (emmatyping) [inline README.md]:
```suggestion
  console (*F12*) for output. You may need to hit "Cancel" on the modal after sending input for output to appear.
```

**Review by emmatyping — APPROVED** [review]
Looks good! One minor suggestion because I found this non-obvious.

Also, I was planning on setting up Github Pages to automatically push the latest `python.*` via https://github.com/marketplace/actions/deploy-to-github-pages so people can just try things out.

File: README.md (hunk 1 above)
Comment (tiran) [reply README.md]: model? did you mean modal?

Comment (tiran) [reply README.md]: I have edited your suggestion.

Comment (emmatyping) [reply README.md]: Yes 😅

**Pull request merged**